src/ui/czi-list.css

```diff
@@ -15,139 +16,35 @@
   counter-reset: none;
   display: flow-root;
   list-style-type: none;
-  margin-block-end: 0;
-  margin-block-start: 0;
   margin: 0;
   padding: 0;
 }
 
-/* bullet indentation */
 .ProseMirror li {
-  /* This assumes the space needed for 3 digits numbers */
-  display: block;
+  display: flex;
   list-style-type: none;
   margin: 0;
   padding: 0;
   position: relative;
   zoom: 1;
 }
 
-.ProseMirror ol {
-  counter-reset: czi-counter;
-  list-style-type: none;
-}
-
-.ProseMirror ol li {
-  counter-increment: czi-counter;
-}
-
 .ProseMirror li > p {
+  flex: 1;
   margin: 0;
-  margin-block-end: 0;
-  margin-block-start: 0;
-  padding-left: 24pt;
 }
 
-/* https://css-tricks.com/numbering-in-style/ */
-.ProseMirror li > p::before {
+/* generic styles for custom list style type. */
+
+/* see https://css-tricks.com/numbering-in-style/ */
+.ProseMirror li::before {
+  box-sizing: border-box;
   color: var(--czi-list-style-color);
-  content: counter(czi-counter) '.';
-  display: inline-block;
-  font-size: 1em;
-  line-height: 1em;
+  content: '';
+  display: block;
+  font-size: 0.95em;
   position: relative;
   text-align: right;
-  top: 0;
-  left: -5px;
-  margin-left: -24pt;
-  width: 24pt;
-}
-
-.ProseMirror ul li > p::before {
-  content: '\2022'; /* disc */
-}
-
-.ProseMirror ol[data-list-style-type='lower-latin'] li > p::before {
-  content: counter(czi-counter, lower-latin) '.';
-}
-
-.ProseMirror ol[data-list-style-type='upper-latin'] li > p::before {
```

Comment:
are we no longer doing `lower-latin`, `upper-latin`, `upper-roman`, `dash`, and `box`?

Reply:
`dash` and `box` are not valid list types.
We do support `upper-latin`, `upper-roman`. These value may be used for cod converted from Google Docs. We don't have the UI that let user pick list style types though.